tests/runtimetest/src/tests.rs

```diff
@@ -307,3 +310,52 @@ pub fn validate_sysctl(spec: &Spec) {
         }
     }
 }
+
+pub fn validate_scheduler_policy(spec: &Spec) {
+    let proc = spec.process().as_ref().unwrap();
+    let sc = proc.scheduler().as_ref().unwrap();
+    println!("schedul is {:?}", spec);
+    let size = mem::size_of::<nc::sched_attr_t>().try_into().unwrap();
+    let mut get_sched_attr = nc::sched_attr_t {
+        size: 0,
+        sched_policy: 0,
+        sched_flags: 0,
+        sched_nice: 0,
+        sched_priority: 0,
+        sched_runtime: 0,
+        sched_deadline: 0,
+        sched_period: 0,
+        sched_util_min: 0,
+        sched_util_max: 0,
+    };
+    unsafe {
+        match nc::sched_getattr(0, &mut get_sched_attr, size, 0) {
```

Comment:
Also, here we are checking the attr of pid 0, while in youki code, we are doing setattr for pid 1 ; is that correct?